tests/testthat/test-renderer2-widerect.R

```diff
@@ -1,26 +1,39 @@
 acontext("geom_widerect")
 
+expect_source <- function(expected){
+  a.list <- getNodeSet(info$html, '//a[@id="a_source_href"]')
+  computed <- if(length(a.list)==0){
+    NULL
+  }else{
+    at.mat <- sapply(a.list, xmlAttrs)
+    at.mat["href",]
+  }
+  expect_identical(computed, expected)
+}
+
 recommendation <- data.frame(
   min.C=21,
   max.C=23)
 set.seed(1)
 temp.time <- data.frame(
   time=strptime(paste0("2015-10-", 1:31), "%Y-%m-%d"),
   temp.C=rnorm(31))
-
-viz <- list(
+viz <- animint(
   gg=ggplot()+
     theme_bw()+
     theme_animint(height=200, width=2000)+
-    geom_widerect(aes(ymin=min.C, ymax=max.C),
-                  color=NA,
-                  fill="grey",
-                  data=recommendation)+
-    geom_line(aes(time, temp.C),
-              data=temp.time)
-  )
+    geom_widerect(aes(
+      ymin=min.C, ymax=max.C),
+      color=NA,
+      fill="grey",
+      data=recommendation)+
+    geom_line(aes(
+      time, temp.C),
+      data=temp.time)
+)
 
 info <- animint2HTML(viz)
+expect_source(NULL)

```

Comment:
this test makes sure that no source `<a>` tag is drawn, when the user does not specify any source option.